README.md

```diff
@@ -3,14 +3,14 @@ This has useful links to help you get started with contributing to open sources.
 
 ### This repository contains
 
-1. Twitter Accounts which tweets about beginner friendly issues.
-2. List of Website which curates beginner friendly issues.
+1. [Twitter Accounts which tweets about beginner friendly issues](https://github.com/tapasweni-pathak/Getting-Started-With-Contributing-to-Open-Sources/blob/master/Twitter.md)
```

Comment:
this can be shortened to just `Twitter.md`. It will act as a relative link and should parse correctly.